meta-resin-common/classes/kernel-resin.bbclass

```diff
@@ -676,7 +676,7 @@ do_kernel_resin_injectconfig[dirs] += "${WORKDIR} ${B}"
 # Reconfigure kernel after we inject resin configs
 #
 do_kernel_resin_reconfigure() {
-    eval ${KERNEL_CONFIG_COMMAND}
```

Comment:
@floion how is this going to work on older bitbake versions? Do all the versions we support have the wanted result?

Reply:
@agherzan I did a quick check and krogoth has this change. Most likely all the other versions have it too